src/app/components/info-card/info-card.tsx

```diff
@@ -78,13 +80,16 @@ export function InfoCardAssetValue({ value, fiatValue, symbol, icon }: InfoCardA
         <Text
           fontSize="24px"
           fontWeight="500"
-          mb="4px"
           lineHeight="36px"
           data-testid={SharedComponentsSelectors.InfoCardAssetValue}
         >
           {value} {symbol}
         </Text>
-        <Text fontSize="12px">~ {fiatValue} USD</Text>
+        {fiatValue && (
+          <Text fontSize="12px" mt="4px">
+            ~ {i18nFormatCurrency(fiatValue)} {fiatValue.symbol}
```

Comment:
For now, `i18nFormatCurrency` only works with USD, but it adds the $ sign so no symbol is needed.

Either remove symbol, or use `formatMoney`

Reply:
hm, there is both symbol and $ sign in the design
![Screenshot 2023-03-25 at 22 17 12](https://user-images.githubusercontent.com/46521087/227734563-1d212b8f-4610-404d-b089-e13bf823ca07.png)